content/vault/v1.20.x/content/partials/telemetry-metrics/vault/raft/thread/fsm/saturation.mdx

```diff
@@ -0,0 +1,3 @@
+Metric type | Value       | Description
+----------- | ----------- | -----------
+gauge       | percentage  | An approximate measurement of the proportion of time the Raft FSM goroutine is busy and unavailable to accept new work
```

Comment:
@include '/vault/docs/global/alert.mdx'
```suggestion
gauge       | percentage  | Approximate proportion of time the raft FSM goroutine could not accept new work
```

Do we know more about what constitutes "busy"? And does the metric also take into account situations where the routine is down for some reason or there's a networking error of some kind? Trying to figure out if there's enough additional information we might want to include as an elaboration below the table.